ci/azure-pipelines.yml

```diff
@@ -153,8 +159,18 @@ stages:
             workingDirectory: $(System.DefaultWorkingDirectory)/ci/vm/
 
           - bash: |
+              set -x
```

Comment:
You may need to add `set -eE` - otherwise the script may proceed even if some commands inside failed

Reply:
Unless it's intended behaviour.

Reply:
It is intended, I even `set +e` down the line.